### PR TITLE
Remove unused nodes

### PR DESCRIPTION
**Pull request by moritzdannhauer**
Done!

### CODE REVIEW

**Comment by dcwhite** [comment]
@moritzdannhauer Please use the new module factory config file approach!
BTW I'm not @Dan, but @dcwhite.

**Comment by moritzdannhauer** [comment]
How about now?

File: src/Modules/Factory/Config/RemoveUnusedNodes.module [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "module": {
+    "name": "RemoveUnusedNodes",
+    "namespace": "Fields",
+    "status": "In progress: needs more testing",
+    "description": "Removes meshnodes that are not connected in any mesh
```
Comment (dcwhite) [inline src/Modules/Factory/Config/RemoveUnusedNodes.module]: @moritzdannhauer This is breaking the build. Newlines in these descriptions are not supported.

**Comment by dcwhite** [comment]
Can you check that your code is building? Travis gets a linking error.

**Comment by dcwhite** [comment]
@moritzdannhauer Similar fix needed at the algo level--count needs increasing:
```
[0;32m[ RUN      ] [mAlgorithmLayerTests.ListAllAlgorithms
algorithm factory size: 82
/home/travis/build/SCIInstitute/SCIRun/src/Core/Algorithms/Factory/Tests/AlgorithmLayerTests.cc:45: Failure
```

**Pull request merged**